# Article_outline.md

## Introduction

1. Crowd sourced genome analysis
	* Raw data publicly available
	* Analysis metadata provided
	* Open science data sharing

2. Ash dieback and the OADB

3. Frameworks for crowd sourcing


## Results and Discussion

Two major sections
1) Github data organization and pointers for future project developers
2) Ash die back data description


### Database structure
Section 1 points:

1. Describe folder structure
 	* Organism -> strain -> subfolders

	* assemblies, annotations, 	blasts,
	raw data - reads, read quality assesments

2. Metadata and templates provided
	* .README files
	* .info files
	* other metadata files



 + Benefits similar folder structure for other projects intended on github
		Specificities of Sepecies in question would have to taken in to account.
+ Advantages of the github
		all activity is documented and
		contributions can be acknowledged better and problem tracking could be easier, with transparent anlysis.
+ With the analysis strcture and guidance available
		these could be tremenedouls help to setup future genome analysis of quick crowd sourcing and anlysis pipelines could setup with minimal implementation times.


### Description of data
Section 2 points:

Explanation of data sets:

1. genomes available

2. transcriptomes available

3. annotations available

User inpterface, search and download options

Description of data presented in the current database and analyses on blog page

Take few points from 1year report and any further ideas

Utility of the data to the community of researchers and discuss salient features of the version controlled database system.

## Summary

## Figures and tables
1. Layout of blog and the github and folder structure

2. Datasets available

3. Example of browsing data


